src/test/java/org/opensearch/ad/feature/SearchFeatureDaoTests.java

```diff
@@ -265,7 +257,10 @@ public void getLatestDataTime_returnExpectedToListener() {
             return null;
         }).when(client).search(eq(searchRequest), any(ActionListener.class));
 
-        when(ParseUtils.getLatestDataTime(eq(searchResponse))).thenReturn(Optional.of(epochTime));
+        InternalMax maxAgg = new InternalMax(CommonName.AGG_NAME_MAX_TIME, epochTime, DocValueFormat.RAW, emptyMap());
```

Comment:
just wondering why does there need to be this new code added here

Reply:
This is part of phasing out PowerMock , the reference is https://github.com/opensearch-project/anomaly-detection/pull/985 (which was never backported saly), thanks @amitgalitz for looking

Reply:
I see, thank you for making this change!